pypit/arms.py

```diff
@@ -53,10 +66,32 @@ def ARMS(spectrograph, fitstbl, setup_dict):
     basenames = [None]*numsci  # For fluxing at the very end
 
     # Spectrometer class
-    spectro_class = spectro_utils.load_spec_class(spectrograph=spectrograph)
+    if spectrograph is None:
+        # Set spectrograph from FITS table instrument header
+        # keyword.
+        if par is not None and par['rdx']['spectrograph'] != fitstbl['instrume'][0]:
+            msgs.error('Specified spectrograph does not match instrument in the fits table!')
+        _spectrograph = load_spectrograph(spectrograph=fitstbl['instrume'][0])
+    elif isinstance(spectrograph, basestring):
+        _spectrograph = load_spectrograph(spectrograph=spectrograph)
+    elif isinstance(spectrograph, Spectrograph):
+        _spectrograph = spectrograph
+    else:
+        raise TypeError('Could not instantiate Spectrograph!')
+
+    # Instantiate the parameters
+    _par = _spectrograph.default_pypit_par() if par is None else par
+    if not isinstance(_par, pypitpar.PypitPar):
```

Comment:
I suppose the renaming of PypitPar will change to PypeItPar (probably a future PR, right?)

Reply:
Yep.  Will do this before 0.9.